src/test/java/fi/otavanopisto/kuntaapi/test/AbstractIntegrationTest.java

```diff
@@ -62,8 +64,9 @@ protected void waitApiListCount(String path, int count) throws InterruptedExcept
           fail(String.format("Timeout waiting for %s to have count %d", path, count));
         }
         
-        if ((counter % 30) == 0) {
-          logger.info(String.format("... still waiting %d items, current count %d", count, listCount));
+        if ((counter % 5) == 0) {
+          System.out.println(String.format("... still waiting %d items, current count %d", count, listCount));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)